docs/modules/pbmarkdup.md

```diff
@@ -0,0 +1,15 @@
+---
+Name: pbmarkdup
+URL: https://github.com/PacificBiosciences/pbmarkdup
+Description: >
+  pbmarkdup takes one or multiple sequencing chips of an amplified libray as
+  HiFi reads and marks or removes duplicates.
+---
+
+The pbmarkdup module parses results generated by
+[pbmarkdup](https://github.com/PacificBiosciences/pbmarkdup), and adds the
+**% Unique Molecules** and **%Duplicate Reads** to the General Statistics
+table.
+
+This module assumes that pbmarkdup has been run for every sample, and that
+the pbmarkup log file is named `sample.pbmarkdup.log`.
```

Comment:
Is this a fair assumption? ie. Is it the default from the tool or something? If the filename is always determined by the user then it's usually better to search file contents instead. If possible I try not to force users to run upstream tools in a specific manner.

Reply:
Looking at the [docs](https://github.com/PacificBiosciences/pbmarkdup#can-i-print-summary-information) I think that this is probably a stdout thing. So maybe better to search for file content instead.

Reply:
I've removed this from the documentation, as we are now using the file header